app/src/main/java/nerd/tuxmobil/fahrplan/congress/models/Meta.kt

```diff
@@ -1,13 +1,18 @@
 package nerd.tuxmobil.fahrplan.congress.models
 
 import info.metadude.android.eventfahrplan.database.contract.FahrplanContract.MetasTable
+import org.threeten.bp.ZoneId
 
 data class Meta(
 
         @Deprecated("To be removed. Access from AppRepository only. Left here only for data transfer.")
         var eTag: String = "",
         var numDays: Int = MetasTable.Defaults.NUM_DAYS_DEFAULT,
         var subtitle: String = "",
+        /**
+         * Valid time zone ID or `null` if validation failed.
```

Comment:
It doesn't feel like this comment adds any value.

Reply:
I don't have strong feelings about it. :broken_heart: Removed in 0714fdd0771a25652917a74919f1ab435a3da221.